### PR TITLE
added note section

### PR DESCRIPTION
**Pull request by Hamzaisongit**
mainly worked on UI, and synced it with dexie.. didn't hamper any of the previous features,
 yet to work on content script, it doesn't do anything yet

### CODE REVIEW

File: src/pages/Background/index.js [hunk 1]
```diff
@@ -1,2 +1,21 @@
+import dexieStore, { db } from "../../Dexie/DexieStore";
+
 console.log('This is the background page.');
-console.log('Put the background scripts here.');
+
+
+
+
+
+// db.symbols.bulkAdd([
```
Comment (Hamzaisongit) [inline src/pages/Background/index.js:7]: for adding dummy data in db, just for testing the ui

File: src/Dexie/DexieStore.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    setItem: setFunc,
+    db
+}
+
+async function getFunc(itemName) {
+    switch (itemName) {
+        case 'notes':
+            const notes = await db.notes.toArray()
+            return notes
+            break;
```
Comment (krishna-404) [inline src/Dexie/DexieStore.js:30]: you can replace this with `return db.notes.Array()`

File: src/Dexie/DexieStore.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+        case 'currentNotes':
+            const { activeSymbol } = await chrome.storage.local.get(['activeSymbol']) || null
+            const currentNotes = await db.notes.where('symId').equals(activeSymbol.symId).toArray()
+            return currentNotes
+            break;
```
Comment (krishna-404) [inline src/Dexie/DexieStore.js:36]: can replace this with `return db.notes.where('symId').equals(activeSymbol.symId).toArray()`

File: src/Dexie/DexieStore.js [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+            const toBeDeletedNotes = await db.notes.where('symId').equals(activeSymbol.symId).toArray()
+
+            await db.notes.bulkDelete([...toBeDeletedNotes.map((obj) => {
+                return obj.noteId
+            })])
+
+            await db.notes.bulkPut([...payload.notes])
+            return
+            break;
```
Comment (krishna-404) [inline src/Dexie/DexieStore.js:57]: didnt understand whats happening here.

File: README.md [hunk 5]
```diff
@@ -1,150 +1,52 @@
...
+# Finfriend - notes feature demo 
 
-[![npm](https://img.shields.io/npm/v/chrome-extension-boilerplate-react)](https://www.npmjs.com/package/chrome-extension-boilerplate-react)
-[![npm-download](https://img.shields.io/npm/dw/chrome-extension-boilerplate-react)](https://www.npmjs.com/package/chrome-extension-boilerplate-react)
-[![npm](https://img.shields.io/npm/dm/chrome-extension-boilerplate-react)](https://www.npmjs.com/package/chrome-extension-boilerplate-react)
+Originally forked from : https://github.com/teziapp/clipboard-extension
 
-## Announcements
+**Working Features as of now :**
+- All previous features
```
Comment (krishna-404) [inline README.md:8]: A Readme file should always be timeless. There is no previous/before in a readme file.

File: serverHomePage/home.html
Comment (krishna-404) [inline serverHomePage/home.html:1]: What is this?

**Review by krishna-404 — CHANGES_REQUESTED** [review]
Too big update. Please break down into micro-PRs as discussed.

**Pull request closed without merging**